includes/Maps_MappingService.php

```diff
@@ -248,6 +248,17 @@ public function addResourceModules( $modules ) {
 	}
 
 	/**
+	 * Adds the layer dependencies.
+	 *
+	 * @param array $dependencies
+	 */
+	public function addLayerDependencies( array $dependencies ) {
```

Comment:
> already exists and does something different.

Of course it does...

![facepalm](https://cloud.githubusercontent.com/assets/146040/18476292/badf45b8-79c9-11e6-89ac-c55bfd75dd2d.jpeg)

`addHtmlDependency` and `addHtmlDependencies` seem like reasonable names for these. The existing `addDependencies` could be split into `addDependenciesToParser` and `addDependenciesToOutput`